lua/super-commit/window/open.lua

```diff
@@ -1,32 +1,22 @@
-local buf_num = require('super-commit/buffer/open').number
-
 local M = {}
+M.id = {}
 
 function M.single_vertical()
   vim.cmd('vsplit')
-  vim.cmd('enew')
-end
-
-local function set_current_number(k)
-  buf_num[k] =  vim.api.nvim_get_current_buf();
 end
 
 function M.setup()
   vim.cmd('rightbelow vsplit')
   vim.cmd('wincmd l')
-  vim.cmd('enew')
-  set_current_number("diff")
+  M.id["diff"] = vim.api.nvim_get_current_win()
   vim.cmd('wincmd h')
   vim.cmd('rightbelow split')
   vim.cmd('wincmd j')
-  vim.cmd('enew')
-  set_current_number("status")
+  M.id["status"] = vim.api.nvim_get_current_win()
   vim.cmd('wincmd l')
   vim.cmd('rightbelow split')
   vim.cmd('wincmd j')
-  vim.cmd('enew')
-  set_current_number("filelist")
-
+  M.id["filelist"] = vim.api.nvim_get_current_win()
```

Comment:
What about using "key" as an argument for the same initial function?

```suggestion
  function M.setup()
    init("diff")
    init("status")
    init("filelist")
```

```
local win_cmds = {
  diff = {
    "rightbelow split",
    "wincmd l",
  },
  status = {
    "wincmd h",
    "rightbelow split",
    "wincmd j",
  },
  filelist = {
    "wincmd l",
    "rightbelow split",
    "wincmd j",
  },
}

local function init(key)
  for _,v in win_cmd[key] do
    vim.cmd(v)
  end
  M.id[key] = vim.api.nvim_get_current_win()
end
end
```

Reply:
Thank you for your suggestion.
I modified it in 2ed21f97bed00595f6bc59b5e6f715600a5c40b7